docs/dev/analysis/features/cht-axis-title.rst

```diff
@@ -0,0 +1,231 @@
+        <xsd:element name="crossesAt" type="CT_Double"/>
+      </xsd:choice>
+    </xsd:sequence>
+  </xsd:group>
+
+  <xsd:complexType name="CT_Title">
+    <xsd:sequence>
+      <xsd:element name="tx" type="CT_Tx" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="layout" type="CT_Layout" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="overlay" type="CT_Boolean" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="spPr" type="a:CT_ShapeProperties" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="txPr" type="a:CT_TextBody" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="extLst" type="CT_ExtensionList" minOccurs="0" maxOccurs="1"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="CT_Tx">
+    <xsd:sequence>
+      <xsd:choice>
+        <xsd:element name="strRef" type="CT_StrRef"/>
+        <xsd:element name="rich"   type="a:CT_TextBody"/>
+      </xsd:choice>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="CT_Layout">
+    <xsd:sequence>
+      <xsd:element name="manualLayout" type="CT_ManualLayout"  minOccurs="0"/>
+      <xsd:element name="extLst"       type="CT_ExtensionList" minOccurs="0"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="CT_ShapeProperties">
+    <xsd:sequence>
+      <xsd:element name="xfrm"               type="CT_Transform2D"            minOccurs="0"/>
+      <xsd:group   ref="EG_Geometry"                                          minOccurs="0"/>
+      <xsd:group   ref="EG_FillProperties"                                    minOccurs="0"/>
+      <xsd:element name="ln"                 type="CT_LineProperties"         minOccurs="0"/>
+      <xsd:group   ref="EG_EffectProperties"                                  minOccurs="0"/>
+      <xsd:element name="scene3d"            type="CT_Scene3D"                minOccurs="0"/>
+      <xsd:element name="sp3d"               type="CT_Shape3D"                minOccurs="0"/>
+      <xsd:element name="extLst"             type="CT_OfficeArtExtensionList" minOccurs="0"/>
+    </xsd:sequence>
+    <xsd:attribute name="bwMode"             type="ST_BlackWhiteMode"         use="optional"/>
+  </xsd:complexType>
+
+  <xsd:complexType name="CT_TextBody">
+    <xsd:sequence>
+      <xsd:element name="bodyPr"   type="CT_TextBodyProperties"/>
+      <xsd:element name="lstStyle" type="CT_TextListStyle"      minOccurs="0"/>
+      <xsd:element name="p"        type="CT_TextParagraph"      maxOccurs="unbounded"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+
+XML specimens
+-------------
+
+.. highlight:: xml
+
+Minimal working XML for a chart with axis titles::
+
+  <?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+  <c:chartSpace xmlns:c="http://schemas.openxmlformats.org/drawingml/2006/chart"
+                xmlns:a="http://schemas.openxmlformats.org/drawingml/2006/main"
+                xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships">
+    <c:chart>
```

Comment:
There's an element that can go right in here, roughly just above c:plotArea, called c:autoTitleDeleted (a boolean I believe). There are some examples on this analysis page that have it True (1), I usually see it False (0) when I come across it for some reason. (It probably makes sense to find the c:chart schema definition on another analysis page and paste it into the schema excerpt so we can see where the two fit with respect to each other. This one has it: https://github.com/scanny/python-pptx/blob/master/docs/dev/analysis/features/cht-chart.rst)

Anyway, we should figure out exactly what it does. The spec might have something to say about it:
https://github.com/scanny/python-pptx/blob/master/spec/ISO-IEC-29500-1/ISO-IEC-29500-1.pdf

My suspicion is that it overrides a title definition, like a visible/not visible flag. Assuming that's so, we'll need to deal with it. A couple design issues come to mind:

* Should there be (and does the MS API have) a separate property for Title.visible = True/False?
* When True is assigned to Axis.has_title, should we clear this flag if it's set?
* When False is assigned to Axis.has_title, where a title is defined (c:title element is present), should we delete the c:title element or just set c:autoTitleDeleted to True?

We'll probably also need to validate the PowerPoint behavior that when c:autoTitleDeleted is set True when a title is defined, that in fact no title appears (and it behaves exactly like when no title is defined as far as spacing and so forth).

The simplest scenario would be that it's just effectively redundant with the presence of a c:title element and we pretty much just delete it whenever we encounter it; but I suspect it's there for a reason we haven't discovered yet :)

I do note that there is a c:strRef inside c:tx, an alternative to c:rich, meaning the value can be defined in the Excel chart data (basically a worksheet cell address). That relationship would be lost if the c:title element was deleted, indicating to me there's a useful case for "hiding" the title rather than having to delete it if you don't want to show it just now.

Over the weekend I happened to be experimenting with Chart title a bit. I noticed that if you add a chart title, put some text in it, and then select No Title (all this in PowerPoint), then if you select Chart Title again, your text is gone. I believe I checked the XML with opc-diag and saw that the title element was actually deleted in that process. Anyway, we'll need to suss that out :)